src/Chapter2.hs

```diff
@@ -500,8 +518,12 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
 
+isThird42 :: [Int] -> Bool
+isThird42 [] = False
+isThird42 [x] = False
+isThird42 [x, y] = False
```

Comment:
If you put the first case on the first line, you can just pattern match on `_` in the last case, as you are interested only in capturing one specific pattern